package.json

```diff
@@ -41,14 +42,11 @@
     "ycsb": "node ./benchmark/ycsb.js  run -P ./benchmark/workloada -p table=usertable -p cloudspanner.instance=ycsb-instance -p operationcount=100 -p cloudspanner.database=ycsb",
     "fix": "eslint --fix '**/*.js'",
     "clean": "gts clean",
-    "compile": "tsc -p . && cp -r src/v1 build/src && cp -r protos build && cp test/*.js build/test",
+    "compile": "tsc -p . && cp -r src/v1 build/src && cp -r protos build && cp -r proto build && cp -r test/*.js build/test",
     "prepare": "npm run compile",
     "pretest": "npm run compile",
     "presystem-test": "npm run compile",
-    "proto": "npm run proto:spanner && npm run proto:spanner_instance_admin && npm run proto:spanner_database_admin",
-    "proto:spanner": "mkdir -p proto && pbjs -t static-module -w commonjs -p node_modules/google-proto-files google/spanner/v1/spanner.proto | pbts -o proto/spanner.d.ts -",
-    "proto:spanner_instance_admin": "pbjs -t static-module -w commonjs -p node_modules/google-proto-files google/spanner/admin/instance/v1/spanner_instance_admin.proto | pbts -o proto/spanner_instance_admin.d.ts -",
-    "proto:spanner_database_admin": "pbjs -t static-module -w commonjs -p node_modules/google-proto-files google/spanner/admin/database/v1/spanner_database_admin.proto | pbts -o proto/spanner_database_admin.d.ts -",
+    "proto": "mkdir -p proto && pbjs -t static-module -w commonjs -p node_modules/google-proto-files google/spanner/v1/spanner.proto google/spanner/admin/instance/v1/spanner_instance_admin.proto google/spanner/admin/database/v1/spanner_database_admin.proto | pbts -o proto/spanner.d.ts -",
```

Comment:
Any reason why we couldn't use the local proto files instead of `google-proto-files`?

Reply:
Yeah, I get bugs like: `Error: no such type: google.iam.v1.SetIamPolicyRequest` which basically means that the `/iam/` folder can't be found within `protos/`. The `protos/google/` folder, if I'm correctly understanding the error, doesn't have every type or doesn't have them fully mapped out 😅

Reply:
Well then!

@jkwlui any reason why synth tool doesn't give us IAM protos?

Reply:
@alexander-fenster